internal/services/datafactory/data_factory_integration_runtime_self_hosted_resource_test.go

```diff
@@ -73,7 +73,11 @@ resource "azurerm_data_factory_integration_runtime_self_hosted" "test" {
 func (IntegrationRuntimeSelfHostedResource) rbac(data acceptance.TestData) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
Needing this suggests that a resource is not being correctly deleted. Can we look into what's not being removed and see if there's a delete bug?